neural_compressor/conf/config.py

```diff
@@ -789,7 +789,8 @@ def percent_to_float(data):
         'framework': And(str, lambda s: s in list(FRAMEWORKS.keys()) + ['NA']),
         Optional('inputs', default=[]): And(Or(str, list), Use(input_to_list)),
         Optional('outputs', default=[]): And(Or(str, list), Use(input_to_list)),
- 
+        Optional('model_inplace', default=False): And(bool, lambda s: s in [True, False]),
```

Comment:
Can we put under quantization/? framework is general and is not supposed to have quantization specific implementation.

Reply:
It is under model field， not framework.